### ISSUE TITLE
AttributeError when running example in README.

### ISSUE THREAD

**Issue by robertnishihara** [Yard1/ray-skorch#15]
I tried running the first example (with numpy/pandas) and got the following error.

Using
ray 1.9.1
torch 1.9.0

I tried again with torch 1.10.0 and got the same error.

```
---------------------------------------------------------------------------
RayTaskError                              Traceback (most recent call last)
<ipython-input-6-6ef515ea9c7a> in <module>
      2 
      3 # predict_proba returns a ray.data.Dataset
----> 4 y_proba = net.predict_proba(X).to_pandas()
      5 print(y_proba)

~/opt/anaconda3/lib/python3.7/site-packages/ray_skorch/base.py in predict_proba(self, X)
    838                 },
    839                 dataset={"dataset": ray_dataset.X},
--> 840                 callbacks=callbacks,
    841             )
    842         datasets: List[ray.data.Dataset] = [

~/opt/anaconda3/lib/python3.7/site-packages/ray/train/trainer.py in run(self, train_func, config, callbacks, dataset, checkpoint, checkpoint_strategy)
    277                 checkpoint=checkpoint,
    278                 checkpoint_strategy=checkpoint_strategy,
--> 279                 run_dir=self.latest_run_dir,
    280             )
    281             for intermediate_result in iterator:

~/opt/anaconda3/lib/python3.7/site-packages/ray/train/trainer.py in __init__(self, backend_executor_actor, backend_config, train_func, run_dir, dataset, checkpoint_manager, checkpoint, checkpoint_strategy)
    587             dataset=dataset,
    588             checkpoint=checkpoint,
--> 589             checkpoint_strategy=checkpoint_strategy)
    590 
    591         self._final_results = None

~/opt/anaconda3/lib/python3.7/site-packages/ray/train/trainer.py in _start_training(self, train_func, run_dir, dataset, checkpoint, checkpoint_strategy, latest_checkpoint_id)
    608         checkpoint_dict = self._checkpoint_manager._load_checkpoint(checkpoint)
    609         self._run_with_error_handling(
--> 610             lambda: ray.get(self._backend_executor_actor.start_training.remote(
    611                 train_func=train_func,
    612                 run_dir=run_dir,

~/opt/anaconda3/lib/python3.7/site-packages/ray/train/trainer.py in _run_with_error_handling(self, func)
    618     def _run_with_error_handling(self, func: Callable):
    619         try:
--> 620             return func()
    621         except TrainingWorkerError:
    622             # Workers have already been restarted.

~/opt/anaconda3/lib/python3.7/site-packages/ray/train/trainer.py in <lambda>()
    612                 run_dir=run_dir,
    613                 dataset=dataset,
--> 614                 checkpoint=checkpoint_dict
    615             ))
    616         )

~/opt/anaconda3/lib/python3.7/site-packages/ray/_private/client_mode_hook.py in wrapper(*args, **kwargs)
    103             if func.__name__ != "init" or is_client_mode_enabled_by_default:
    104                 return getattr(ray, func.__name__)(*args, **kwargs)
--> 105         return func(*args, **kwargs)
    106 
    107     return wrapper

~/opt/anaconda3/lib/python3.7/site-packages/ray/worker.py in get(object_refs, timeout)
   1711                     worker.core_worker.dump_object_store_memory_usage()
   1712                 if isinstance(value, RayTaskError):
-> 1713                     raise value.as_instanceof_cause()
   1714                 else:
   1715                     raise value

RayTaskError: ray::BackendExecutor.start_training() (pid=62623, ip=127.0.0.1, repr=<ray.train.backend.BackendExecutor object at 0x7f8c582aad50>)
  At least one of the input arguments for this task could not be computed:
ray.exceptions.RaySystemError: System error: Can't get attribute 'MyModule' on <module '__main__' from '/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/ray/workers/default_worker.py'>
traceback: Traceback (most recent call last):
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/ray/serialization.py", line 281, in deserialize_objects
    obj = self._deserialize_object(data, metadata, object_ref)
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/ray/serialization.py", line 194, in _deserialize_object
    return self._deserialize_msgpack_data(data, metadata_fields)
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/ray/serialization.py", line 172, in _deserialize_msgpack_data
    python_objects = self._deserialize_pickle5_data(pickle5_data)
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/ray/serialization.py", line 162, in _deserialize_pickle5_data
    obj = pickle.loads(in_band)
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/skorch/net.py", line 2064, in __setstate__
    cuda_attrs = torch.load(f, **load_kwargs)
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/torch/serialization.py", line 607, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/torch/serialization.py", line 882, in _load
    result = unpickler.load()
  File "/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/torch/serialization.py", line 875, in find_class
    return super().find_class(mod_name, name)
AttributeError: Can't get attribute 'MyModule' on <module '__main__' from '/Users/rkn/opt/anaconda3/lib/python3.7/site-packages/ray/workers/default_worker.py'>
```

**Comment by Yard1** [comment]
Thanks for the report @robertnishihara, this will be fixed through https://github.com/Yard1/ray-skorch/pull/16

**Comment by Yard1** [comment]
Fixed in new release 0.0.2.
